app/admin/companies.rb

```diff
@@ -7,7 +7,7 @@
   config.batch_actions = false
 
   publishable_scopes
-  publishable_sidebar only: :show
+  publishable_resource_sidebar
```

Comment:
We display this sidebar on `show` pages only, it doesn't make sense for collection (we have bulk action for that).